source/auth/auth.md

```diff
@@ -1261,6 +1266,59 @@ OIDC access token from that file. The driver MUST use the contents of that file
 Drivers MAY implement the AWS provider so that it conforms to the function signature of the
 [OIDC Callback](#oidc-callback) to prevent having to re-implement the AWS provider logic in the OIDC prose tests.
 
+**Azure**
```

Comment:
It seems that most of this section describes the same functionality as [the Azure steps](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#L697) in client side encryption. Could we use the same language across both specs? If there is a way to refer from one section to the other, then this would help drivers avoid code duplication. (I think the description here is less language-specific, which is better, but perhaps the timeouts and error handling should be specified?)

Reply:
That section differs in that it explicitly does not support multiple identities, and is not consistent with how API requests are described in this document.

Reply:
Fair enough, though specifying that the server error message should be propagated into our error seems like a nice convenience.

Is it correct that the timeout logic is now implied by the use of an internal callback, and we would use the `callbackTimeoutMS` provided to this Azure machine callback? Will that be an acceptable timeout within which to complete this request?

Reply:
Updated